charts/stable/standardnotes-server/values.yaml

```diff
@@ -0,0 +1,285 @@
+image:
+  pullPolicy: IfNotPresent
+  repository: standardnotes/server
+  tag: latest@sha256:cbdc5e3e6d7212eeaec86898e6ba944b1678349262d3da7215a246d832471391
```

Comment:
is there a versioned tag?

Reply:
The tags look like checksums, I don't see any active ones with versions.
https://hub.docker.com/r/standardnotes/server/tags